h2o-genmodel/src/main/java/hex/genmodel/ModelMojoReader.java

```diff
@@ -240,6 +243,64 @@ else if (line.equals("[columns]")) {
     return domains;
   }
 
+  Map<String, Map<String, int[]>> parseTargetEncodingMap(BufferedReader source) throws IOException {
+    Map<String, Map<String, int[]>> encodingMap = new HashMap<>();
+    Map<String, int[]> encodingsForColumn = null;
+    String sectionName = null;
+    try {
+      String line;
+
+      while (true) {
+        line = source.readLine();
+        if (line == null) { // EOF
+          encodingMap.put(sectionName, encodingsForColumn);
+          break;
+        }
+        line = line.trim();
+        if(sectionName == null) {
+          sectionName = matchNewSection(line);
+          encodingsForColumn = new HashMap<>();
+        } else { 
+          String matchResult = matchNewSection(line);
+          if(matchResult!= null) {
+            encodingMap.put(sectionName, encodingsForColumn);
+            encodingsForColumn = new HashMap<>();
+            sectionName = matchResult;
+            continue;
```

Comment:
continue replacable with else {}, would be more readable

Reply:
@honzasterba subjective. It is safer as I'm explicitly stating that I want to start next iteration in the enclosing `while` loop and reader/developer don't need to go further and make sure that there is nothing that could be executed after `else` block is closed.